src/components/tx-flow/flows/ChangeThreshold/ReviewChangeThreshold.tsx

```diff
@@ -1,69 +1,40 @@
+import { ChangeThresholdFlowFieldNames } from '@/components/tx-flow/flows/ChangeThreshold'
+import type { ChangeThresholdFlowProps } from '@/components/tx-flow/flows/ChangeThreshold'
 
-const ReviewChangeThreshold = () => {
+const ReviewChangeThreshold = ({ params }: { params: ChangeThresholdFlowProps }) => {
   const { safe } = useSafeInfo()
-  const [selectedThreshold, setSelectedThreshold] = useState<number>(safe.threshold)
-  const [isChanged, setChanged] = useState<boolean>(false)
-  const isSameThreshold = selectedThreshold === safe.threshold
+  const newThreshold = params[ChangeThresholdFlowFieldNames.threshold]
 
   const { setSafeTx, setSafeTxError } = useContext(SafeTxContext)
 
   useEffect(() => {
-    if (!selectedThreshold) return
-
-    createUpdateThresholdTx(selectedThreshold).then(setSafeTx).catch(setSafeTxError)
-  }, [selectedThreshold, setSafeTx, setSafeTxError])
-
-  const handleChange = (event: SelectChangeEvent<number>) => {
-    const newThreshold = parseInt(event.target.value.toString())
-    setSelectedThreshold(newThreshold)
-    setChanged(true)
-  }
+    createUpdateThresholdTx(newThreshold).then(setSafeTx).catch(setSafeTxError)
+  }, [newThreshold, setSafeTx, setSafeTxError])
 
   const onChangeThreshold = () => {
     trackEvent({ ...SETTINGS_EVENTS.SETUP.OWNERS, label: safe.owners.length })
-    trackEvent({ ...SETTINGS_EVENTS.SETUP.THRESHOLD, label: selectedThreshold })
+    trackEvent({ ...SETTINGS_EVENTS.SETUP.THRESHOLD, label: newThreshold })
   }
 
   return (
-    <SignOrExecuteForm onSubmit={onChangeThreshold} disableSubmit={isSameThreshold}>
-      <Typography mb={2}>Any transaction will require the confirmation of:</Typography>
-
-      <Grid container direction="row" gap={1} alignItems="center" mb={2}>
-        <Grid item xs={2}>
-          <Select value={selectedThreshold} onChange={handleChange} fullWidth>
-            {safe.owners.map((_, idx) => (
-              <MenuItem key={idx + 1} value={idx + 1}>
-                {idx + 1}
-              </MenuItem>
-            ))}
-          </Select>
-        </Grid>
-
-        <Grid item>
-          <Typography>out of {safe.owners.length} owner(s)</Typography>
-        </Grid>
-      </Grid>
-
-      {isChanged && isSameThreshold ? (
-        <Typography color="error" mb={2}>
-          Current policy is already set to {safe.threshold}
+    <SignOrExecuteForm onSubmit={onChangeThreshold}>
+      <div>
+        <Typography variant="body2" color="text.secondary" mb={0.5}>
+          Any transaction will require the confirmation of:
         </Typography>
-      ) : (
-        <Typography mb={2}>
-          {isChanged ? 'Previous policy was ' : 'Current policy is '}
-          <b>
-            {safe.threshold} out of {safe.owners.length}
-          </b>
-          .
+
+        <Typography>
+          <b>{newThreshold}</b> out of <b>{safe.owners.length} owner(s)</b>
         </Typography>
-      )}
+      </div>
```

Comment:
I suggest to also add a `<Divider className={commonCss.nestedDivider} />` here like in the Design.

Reply:
Included in 6ad56f5eb17e6a2bdecfc3020dd962a409b87544.